.github/workflows/main.yml

```diff
@@ -0,0 +1,123 @@
+name: python-package
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  pkg-install:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+          architecture: x64
+      - name: Cache python modules
+        uses: actions/cache@v1.1.2
+        env:
+          cache-name: cache-pkg-requirements
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-install-${{ env.cache-name }}-${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-install-${{ env.cache-name }}-
+            ${{ runner.os }}-install-
+            ${{ runner.os }}-
+      - name: Install package
+        run: |
+          python -m pip install --upgrade pip
+          pip install --user picamera
+          pip install pyronear
```

Comment:
Since you need a RaspberryPI environment and not  `ubuntu-latest` we will need to use a docker I guess. So for now, I would recommend removing the job

Reply:
Ok I will check that